tests/unit/amo/reducers/test_users.js

```diff
@@ -163,6 +164,28 @@ describe(__filename, () => {
     });
   });
 
+  describe('unloadUserAccount', () => {
+    it('unloads a user from the state given a user ID', () => {
+      const userId = 12345;
+      const username = 'john';
+
+      const user = createUserAccountResponse({ id: userId, username });
+      const prevState = reducer(initialState, loadUserAccount({ user }));
+
```

Comment:
Would it be worth adding some check assertions here for `state.byID[userId]` and `state.byUsername[username]` to verify that the call to `loadUserAccount` successfully loaded the data? Otherwise the assertions at the end could be yielding false positives.

Reply:
Added.